README.md

```diff
@@ -114,6 +114,16 @@ dfx canister call evm_rpc getAuthorized '(variant { RegisterProvider })'
 dfx canister call evm_rpc deauthorize "(principal \"$PRINCIPAL\", variant { RegisterProvider })"
 ```
 
+## Reproducible Builds
+
+The EVM RPC canister supports [reproducible builds](https://internetcomputer.org/docs/current/developer-docs/smart-contracts/test/reproducible-builds):
+
+1. Ensure [Docker](https://www.docker.com/get-started/) is installed on your machine.
+2. Run `scripts/docker-build` in your terminal. 
+3. Run `sha256sum evm_rpc.wasm` on the generated file to view the SHA-256 hash.
```

Comment:
So I tried those instructions on 2 different architectures:

1. In devenv (Ubuntu), where I obtain the hash `a3e2da98a1e93ed24e4f0a05e55aaf06a6fd4540057d503c117a48743d5dae0b`
2. Locally on Mac, where I obtain a **different** hash `934f54ea27983bba6052a9155c8e18243120b1976a3482467333761a2379c019`

which means that currently the build is not reproducible.

```suggestion
4. Run `sha256sum evm_rpc.wasm.gz` on the generated file to view the SHA-256 hash.
```